api/src/main/scala/hmda/api/HmdaApi.scala

```diff
@@ -36,9 +41,27 @@ object HmdaApi
 
   //Start up API Actors
 
-  createSingleLarValidator(system)
+  val supervisor = HmdaSupervisor.createSupervisor(system)
+
+  (supervisor ? FindActorByName(SingleLarValidation.name))
+    .mapTo[ActorRef]
+    .map { actor =>
+      log.info(s"Started validator at ${actor.path}")
+    }
+
+  (supervisor ? FindActorByName(LocalHmdaEventProcessor.name))
+    .mapTo[ActorRef]
+    .map { actor =>
+      log.info(s"Started event processor at ${actor.path}")
+    }
+
+  (supervisor ? FindActorByName(InstitutionPersistence.name))
+    .mapTo[ActorRef]
+    .map { actor =>
+      log.info(s"Started institutions at ${actor.path}")
+    }
+
   createInstitutions(system)
```

Comment:
It looks like we are already creating the Institutions actor above through the supervisor so this one might be just a forgot to remove?